src/pages/filter/dtree/components/query-builder/query-builder-search.tsx

```diff
@@ -53,7 +60,21 @@ export const QueryBuilderSearch = observer(
 
         {!isSubgroupItemSearch && (
           <div className="flex items-center justify-between pl-2 text-grey-blue">
-            <div className="mr-1">
+            {showSwitcher && (
+              <div className="mr-2 flex items-center">
+                <Switch
+                  isChecked={!!isSwitched}
+                  onChange={(e: boolean) => onSwitch && onSwitch(e)}
+                  className="mr-2"
+                />
+
+                <span className="text-sm w-[60px]">
+                  {t(`filter.switcher.${isSwitched ? 'fullWord' : 'partWord'}`)}
```

Comment:
It's not a good idea to change the label when switching the control. It confuses which state we're now switching on or off. Maybe better to call it like this. When it's turned off, we are searching by any entry, and when it's on we're searching by entry the result starts with. 
<img width="301" alt="Screen Shot 2022-05-18 at 02 12 13" src="https://user-images.githubusercontent.com/96476703/168919836-34506b4f-4112-4fc6-8afc-051dc56e2a58.png">

Reply:
Done